### ISSUE TITLE
Re-implementation paper works

### ISSUE THREAD

**Issue by delta0228** [tangtaogo/lidar-nerf#4]
Thank you for sharing your works and efforts!!

I'd try to re-implement your results based on uploaded code.
The output generated by the model trained using kitti360 work fine, which means that it well represents the 3d geometry in the kitti360 dataset.

Unfortunately, however, the result of models that are trained newly using mvl is quite different to that of the pre-trained model that you uploaded.

Indeed, the loss value decreased when I train the model according your introduction. However, it seems like fail to learn nerf information so that fail to generate vehicle-like point cloud swarm.

On the other hand, both loss value and variation of loss value estimated during validation process using uploaded pre-trained model is quite large (maximum value of loss is about 60, and vary 0.01 to 60). 

Despite high loss value, it successfully generates the vehicle-shape point cloud swarm. However, strange thing is that the output in results folder well represent the object shape but one in validations are fail to generate point cloud data that showing object shape.

I guess that It seems like there some issue on loss configuration but not for sure. 

Could you give me some advice for re-implementing your works using mvl dataset?

(Currently, I'm working on V100, with conda virtual environment)

**Comment by tangtaogo** [comment]
Hello, thanks for your feedback. I will double-check the results on mvl, but this may take several days since I have been busy with some work recently. I provide the checking results here as soon as possible.

**Comment by Petros626** [comment]
@tangtaogo is this repo still maintained? I would like to use kitti dataset
